FreeRTOS/iot-reference-nxp-rt1060
# Fix Tera Term link

Description
-----------
The previous link was broken
causing the link verifier to
fail.

Test Steps
-----------
Cloned down link verifier from CI-CD actions repo, ran link verifier

Checklist:
----------


  - Not applicable


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
